src/main/java/App.java

```diff
@@ -0,0 +1,17 @@
+public class App {
+    public static void main(String[] args) {
+        int a = 10;
+        int b = 7;
+        System.out.println(sum(a, b));
+        System.out.println(div(a, b));
+    }
+
+    public static int sum(int a, int b) {
+        return a + b;
+    }
+
+    public static int div(int a, int b) {
+        return a - b;
+    }
```

Comment:
This is the best code i ever seen!
```suggestion
    public static void main(String[] args) {
        int a = 10;
        int b = 7;
        System.out.println(sum(a, b));
        System.out.println(div(a, b));
    }

    public static int sum(int a, int b) {
        return a + b;
    }

    public static int div(int a, int b) {
        return a - b;
    }
    
```